compiler/src/dmd/glue/toir.d

```diff
@@ -127,6 +128,15 @@ struct IRState
         return dmd.funcsem.arrayBoundsCheck(getFunc());
     }
 
+    /**********************
+     * Returns:
+     *    true if do null dereference checking for the current function
+     */
+    bool nullDerefCheck()
+    {
+        return inNullDerefCheckDisable == 0 && dmd.funcsem.nullDerefCheck(getFunc());
+    }
```

Comment:
Having two functions with the same name is confusing. Perhaps make it one function?